WordPress/src/main/java/org/wordpress/android/ui/AppLogViewerActivity.java

```diff
@@ -116,10 +119,21 @@ private void shareAppLog() {
         }
     }
 
+    private void copyAppLogToClipboard() {
+        ClipboardManager clipboard = (ClipboardManager) getSystemService(Context.CLIPBOARD_SERVICE);
```

Comment:
This is obscure, but I recommend adding an exception handler here. [Here's why](https://code.google.com/p/android/issues/detail?id=58043).